### PR TITLE
created scss folder for fancy cards

### PR DESCRIPTION
**Pull request by stephencastaneda**

### CODE REVIEW

File: src/styles/_fancycard.scss [hunk 1]
```diff
@@ -0,0 +1,175 @@
+@import "~bootstrap/scss/bootstrap";
```
Comment (zoeames) [inline src/styles/_fancycard.scss:1]: remove these 2 import statements

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,5 +1,7 @@
 @import "~bootstrap/scss/bootstrap";
 @import "~@fortawesome/fontawesome-free/css/all.min.css";
+@import '_fancycard';
```
Comment (zoeames) [inline src/styles/main.scss:3]: you can remove the _

File: src/styles/_fancycard.scss [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+  padding-top: 0;
+}
+
+.fancy-card .general .more {
+  position: absolute;
+  bottom: 1rem;
+  right: 1rem;
+  font-size: 0.9em;
+}
+
```
Comment (zoeames) [inline src/styles/_fancycard.scss:174]: delete extra blank lines

**Comment by stephencastaneda** [comment]
I made the suggested changes.

**Pull request merged**